benchmark/files/prism.js

```diff
@@ -0,0 +1,909 @@
+/* PrismJS 1.17.1
```

Comment:
What's the deal with this file + min version? Do we need them duplicated here?

Reply:
Just some test files. (I originally also had copies of other files here before adding support for relative file paths.)

Also, the `prism.js` here isn't a copy of the root `prism.js`. Both of these files were generated by the download page and are supposed to represent as decently sized JS library minified and unminified.

Reply:
I removed the file and use the root `prism.js` instead.